enterprise/backup/src/main/java/org/neo4j/backup/OnlineBackupCommand.java

```diff
@@ -203,7 +203,7 @@ public void execute( String[] args ) throws IncorrectUsage, CommandFailed
             try
             {
                 backupService.doIncrementalBackup( address.getHost(), address.getPort(),
-                        destination, timeout, config );
+                        destination, ConsistencyCheck.NONE, timeout, config );
```

Comment:
Shouldn't we use the `check-consistency` argument for this one? Or rather introduce another one for incremental backup if we don't want to change behaviour? This is the main surface of doing a backup so it should be controllable from this surface IMHO.